### PR TITLE
Contributions

### PR DESCRIPTION
**Pull request by Abigailfelix**
More Font-radius /box-shadow was added to the button
Colour of colour changed to Gray and made bolder

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -100,17 +99,21 @@ color: hsl(245, 75%, 52%);
     width: 90%;
     height: 40px ;
     border:none;
-    border-radius: 5px;
+    border-radius: 10px;
     margin:  20px;
+    margin-top: -40px;
```
Comment (jovick-coder) [inline css/style.css]: remove this 
the space between is ok
no need for -40

**Review by jovick-coder — APPROVED** [review]
nice work

**Review by jovick-coder — COMMENTED** [review]
did you create a new css file?

**Comment by Abigailfelix** [comment]
No, I forked it then cloned and edited it.

**Pull request merged**